SPECS/ocaml-ppx-let/ocaml-ppx-let.spec

```diff
@@ -0,0 +1,139 @@
+# OCaml packages not built on i686 since OCaml 5 / Fedora 39.
+ExcludeArch: %{ix86}
+
+Name:           ocaml-ppx-let
+Version:        0.16.0
+Release:        9%{?dist}
+Summary:        Monadic let-bindings for OCaml
```

Comment:
Yup, another missing vendor and distribution.

Reply:
Added it